src/imgui_ex/vcImGuiSimpleWidgets.cpp

```diff
@@ -84,6 +85,40 @@ bool vcIGSW_InputTextWithResize(const char *pLabel, char **ppBuffer, size_t *pBu
   return retVal;
 }
 
+void vcIGSW_FilePicker(vcState *pProgramState, const char *pLabel, char *pBuffer, size_t bufferSize, const char **ppExtensions, size_t numExtensions, bool read, vcFileDialogCallback onChange)
```

Comment:
Consider renaming the `read` parameter to be `loadOnly` - more consistent with the subsequent functions.

Reply:
This was changed to `read` because `loadOnly` is wrong.